src/frontend/components/ResultScrollView.tsx

```diff
@@ -1,6 +1,6 @@
 import clsx from "clsx";
 interface ResultScrollContainerProps {
-  children: JSX.Element | Array<JSX.Element | string>;
+  children: React.ReactNode;
```

Comment:
This is typically the preferred way to type children since it allows for things that are valid 
in JSX like numbers, strings and booleans and is also inclusive of JSX.Element and collections.

Reply:
Yeah, thanks. This is what I was missing the whole time.